### PR TITLE
data: money8203.json

### PR DESCRIPTION
**Pull request by money8203**
## Changes proposed

Updated my profile!

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8030"></a>

### CODE REVIEW

**Comment by sandramsc** [comment]
Hi @money8203  I have left an inline comment with a suggestion.

**Comment by money8203** [comment]
> Hi @money8203 I have left an inline comment with a suggestion, otherwise the file looks good.

Can't see any !? Can you please request suggestions.

File: data/money8203.json [hunk 1]
```diff
@@ -34,6 +34,12 @@
       "url": "https://twitter.com/manishtyagi__",
       "icon": "FaTwitter"
     },
+    {
+      "group": "Socials",
+      "name": "Let's Thread together",
+      "url": "https://threads.net/@manishtyagi__",
+      "icon": "FaThreads"
```
Comment (kumarsonsoff3) [inline data/money8203.json:41]: `FaThreads` is not a valid icon. Please update it with something from [these icons](https://linkfree.io/icons).

File: data/money8203.json [hunk 2]
```diff
@@ -19,11 +19,11 @@
     },
     {
       "icon": "FaLinkedin",
-      "url": "https://linkedin.com/in/manish-tyagi-8b7b22217"
+      "url": "https://linkedin.com/in/manishty"
     },
     {
-      "icon": "FaInstagram",
-      "url": "https://instagram.com/manish_tyagi.__"
+      "icon": "FaThreads",
```
Comment (kumarsonsoff3) [inline data/money8203.json:25]: Also this one!

**Review by sandramsc — CHANGES_REQUESTED** [review]

File: data/money8203.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (sandramsc) [reply data/money8203.json:41]: Made the review couple hours ago only learned how to make it public now...thanks for the notification!

Comment (sandramsc) [reply data/money8203.json:41]: `FaInstagram` might be the closest replacement for this icon key?

File: data/money8203.json (hunk 2 above)
Comment (money8203) [reply data/money8203.json]: Resolved @kumarsonsoff3!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**